src/More/ENSubmitDebugForm.tsx

```diff
@@ -172,13 +156,14 @@ const style = StyleSheet.create({
     height: "100%",
     resizeMode: "cover",
     backgroundColor: Colors.faintGray,
+    paddingBottom: Spacing.medium,
   },
   container: {
     height: "100%",
     justifyContent: "space-between",
     paddingHorizontal: Spacing.medium,
     paddingTop: Spacing.large,
-    backgroundColor: Colors.primaryBackground,
+    backgroundColor: Colors.faintGray,
```

Comment:
Just curious why not use `primaryBackground` here?

Reply:
I was trying to match the styling of the feedback form for consistency.

Reply:
@devinjameson ^^ Let me know if this is a good change to make or not

Reply:
Sure that works!